tests/virt/node/hyperv_support/test_windows_crash_detection.py

```
@@ -0,0 +1,102 @@
+                "vm_name": "windows-crash-detection-vm",
+                "template_labels": WINDOWS_LATEST_LABELS,
+            },
+            marks=pytest.mark.polarion("CNV-15265"),
+        ),
+    ],
+    indirect=True,
+)
+@pytest.mark.special_infra
+@pytest.mark.high_resource_vm
+def test_windows_crash_detection_with_hyperv_panic(
+    enabled_featuregate_scope_function,
```

Ruff (0.15.9)

[warning] Unused function argument: `enabled_featuregate_scope_function`

(ARG001)